internal/istio/errors.go

```diff
@@ -13,6 +13,8 @@ var (
 	ErrFailedToConvertLabelSelector      = errors.New("failed to convert label selector to selector")
 	ErrFailedToGetGatewayByLabelSelector = errors.New("failed to get gateway by label selector")
 
+	ErrFailedToAddOwnerReference = errors.New("failed to add owner reference")
+
```

Comment:
I think if we are doing grouping via empty lines, since empty lines are not self-explanatory there should be comments for each group, in general a least.
Since this is all about istio, I don't think we need that detailed description about errors so let's just remove the implicit grouping here. It's just noise and interrupting the natural flow of reading. WDYT?

Reply:
Done with https://github.com/kyma-project/lifecycle-manager/pull/1349/commits/52c2806a62491fa612a6d09e48c999f672b49f0e